### PR TITLE
update CTA hover states to be consistent with animation and non color…

### LINKED ISSUE #15268: Refresh: Hover states and other animation
- we don't use text color to indicate hover state, any green hover indicators should be over-written
  - remember to over-ride :visited:hover states as well
- the [chasing underline](https://codepen.io/stephaniehobson/pen/XWvwGXX) animation should be applied to all ctas

### PR DESCRIPTION
**Pull request by wen-2018**
… change (fix #15268)

## One-line summary

This PR updates the CTA hover styles for the new home and about-us pages

## Significant changes and points to review

CTA links for new home and about us pages:
- hover states
- hover states for visited links

## Issue / Bugzilla link

https://github.com/mozilla/bedrock/issues/15268

## Testing
http://localhost:8000/
http://localhost:8000/about/

### CODE REVIEW

File: media/css/m24/base.scss [hunk 1]
```diff
@@ -19,6 +19,9 @@
 @use 'theme';
 @use 'transition';
 
+$curve: cubic-bezier(0.5, 0, 0, 1);
+$slow: .8s;
```
Comment (stephaniehobson) [inline media/css/m24/base.scss:23]: Please use the variables in `vars/_animation.scss`

File: media/css/m24/base.scss [hunk 2]
```diff
@@ -89,9 +92,21 @@ a:link {
     font-family: var(--title-font-family);
     font-size: $text-button-xl;
     font-weight: 600;
+    background-image:
+    linear-gradient(to right, $m24-color-black 0, $m24-color-black 33%,transparent 33%, transparent 66%, $m24-color-black 66%, $m24-color-black 100%);
+    background-repeat: no-repeat;
+    background-position:
+        left var(--c, 100%) top 100%;
```
Comment (stephaniehobson) [inline media/css/m24/base.scss:99]: Let's give this variable a more descriptive name. Maybe `--cta-bg-position`?

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
Quick change to use the global variables and then it's good to go 🙂

**Comment by wen-2018** [comment]
@stephaniehobson All feedback addressed now

**Review by stephaniehobson — APPROVED** [review]
R+ 🏃

**Pull request merged**